content/concepts/introduction/overview.md

```diff
@@ -1,160 +1,143 @@
+runtimes and networking environments.
+
+> Discovering and connecting with other peers is a key challenge in P2P networking. In the past,
+> each P2P application had to develop its own solution for this problem, leading to a lack of
+> reusable, well-documented P2P protocols. IPFS looked to existing research and networking
+> applications for inspiration, but found few code implementations that were usable and adaptable.
+> Many of the existing implementations had poor documentation, restrictive licensing, outdated code,
+> no point of contact, were closed source, deprecated, lacked specifications, had unfriendly APIs,
+> or were tightly coupled with specific use cases and not upgradeable. As a result, developers often
+> had to reinvent the wheel each time they needed P2P protocols, rather than being able to reuse
+> existing solutions.
+
+{{< alert icon="" context="info">}}
+libp2p was designed to address the limitations of traditional P2P networking approaches and these
+existing network models, with the goal of enabling the distributed web.
+{{< /alert >}}
+
+## Why libp2p?
+
+There are several reasons to consider using libp2p as a networking layer to create a robust P2P application:
+
+- **Modularlity**: libp2p is designed to be modular, allowing developers to mix and match different components
+  to meet the needs of their particular application. This makes it easy to customize the networking stack
+  to fit the specific requirements of any P2P application.
+
+  - **Transport agnosticism**: libp2p provides a set of specifications that can be adapted to support various
+    [transport protocols](../transports/overview.md), allowing libp2p applications to operate in various runtime
+    and networking environments. This makes it possible to use libp2p in a variety of scenarios, regardless of
+    the underlying transport protocol.
+
+  - **Versatility**: In addition to being transport agnostic, libp2p offers a range of discovery mechanisms,
+    data storage and retrieval patterns, and is also
+    [implemented in many programming languages](https://libp2p.io/implementations/), providing developers
+    with great flexibility when building P2P applications.
+
+- **Security**: libp2p includes [several security features](../security/security-considerations.md),
+  such as peer identity verification using public key cryptography, [encrypted communication](../secure-comm/overview.md) between peers using modern cryptographic algorithms, and protection against network attacks through
+  the use of [mitigation techniques](../security/dos-mitigation.md).
+
+- **Robustness**: libp2p is a robust and reliable networking protocol that is designed to withstand stress,
+  disturbance, and change. Its features and design choices ensure that it is able to function effectively
+  and efficiently in a wide range of environments, and it is able to recover quickly from disruptions or
+  failures.
```

Comment:
Can we combine these two and mention the DoS mitigations techniques as an example of robustness